Aspire Budgeting/Facilities/GoogleDriveManager.swift

```diff
@@ -9,35 +9,46 @@
 import Foundation
 import GoogleAPIClientForREST
 import GoogleSignIn
+import GTMSessionFetcher
 
 class GoogleDriveManager: ObservableObject {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mohitathwani/aspirebudgeting/blob/c2420c127e412148682bc0fe81f219ec96ee8752/Aspire Budgeting/Facilities/GoogleDriveManager.swift#L14"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>